src/materials/ConcreteThermalMoisture.C

```diff
@@ -71,25 +71,34 @@ ConcreteThermalMoisture::validParams()
   params.addParam<MooseEnum>(
       "aggregate_pore_type", aggregate_pore_type, "aggregate pore structure");
 
-  params.addParam<Real>("cement_mass", "cement mass (kg) per m^3");
-  params.addParam<Real>("aggregate_mass", "aggregate mass (kg) per m^3");
+  // WGA - unclear if units for following two params is kg or kg/m^3 - assuming kg
```

Comment:
```suggestion
```
You're correct. We should probably change those parameters to be called `density`, and have them be defined as kg/m^3. The provided numbers would be the same. We can deal with that separately, though.